common.sh

```diff
@@ -41,18 +41,17 @@ function queryGHAPI(){
   echo ${exist}
 }
 
-# to check if the same tag has been used to triggere release pipeline
+# to check if the same _adopt scmReference tag has been used to trigger a release pipeline in the past
 # if yes, wont trigger; if not, do the first time trigger
 function checkPrevious() {
   if [ -f ${WORKSPACE}/tracking ]; then # already have tracking from previous
     expectedTag=$1 #
     trackerTag="$(cut -d '=' -f 2 ${WORKSPACE}/tracking)" # previousSCM=jdk-17.0.5+8_adopt
-    olderTag="$(echo -e "${expectedTag}\n${trackerTag}" | sort -V | head -n1)"
 
-    if [[ "${expectedTag}" == "${olderTag}" || "${expectedTag}" == "${trackerTag}" ]]; then
-        echo "Release tag ${trackerTag} has been used in the current release"
+    if [[ "${expectedTag}" == "${trackerTag}" ]]; then
```

Comment:
Not sure I understand this part.
e.g 
when `expectedTag="jdk-17.0.6"`
trackerTag is "jdk-17.0.5+8_adopt"  (read out from tracking, originally written by `echo previousSCM=$scmReference > ${WORKSPACE}/tracking`) 
this will never be `==` , thus not exit